src/app/game-state/activity.service.ts

```diff
@@ -1207,11 +1213,111 @@ export class ActivityService {
+          else this.characterService.characterState.increaseAttribute(attribute, 2.5);
+          statSum += this.characterService.characterState.attributes[attribute].value;
+        }
+        this.characterService.characterState.money += Math.sqrt(statSum);
+        this.getActivityByType(ActivityType.OddJobs).lastIncome = Math.sqrt(statSum);
+        this.oddJobDays++;
+      },
+      () => {
+        this.characterService.characterState.status.stamina.value -= 250;
+        let lowStat = "strength" as AttributeType;
+        let highStat = "strength" as AttributeType;
+        for (const attribute of ["toughness","speed","intelligence","charisma"] as AttributeType[]){
+          if (this.characterService.characterState.attributes[attribute].value < this.characterService.characterState.attributes[lowStat].value) {
+            lowStat = attribute;
+          }
+          if (this.characterService.characterState.attributes[attribute].value > this.characterService.characterState.attributes[highStat].value) {
+            highStat = attribute;
+          }
+        }
+        let statSum = 0;
+        for (const attribute of ["strength","toughness","speed","intelligence","charisma"] as AttributeType[]){
+          if (attribute === lowStat) this.characterService.characterState.increaseAttribute(attribute, 50);
+          else if (attribute === highStat) this.characterService.characterState.increaseAttribute(attribute, 100);
+          else this.characterService.characterState.increaseAttribute(attribute, 75);
+          statSum += this.characterService.characterState.attributes[attribute].value;
+        }
+        this.characterService.characterState.money += Math.sqrt(statSum) * 5;
+        this.getActivityByType(ActivityType.OddJobs).lastIncome = Math.sqrt(statSum) * 5;
+        this.oddJobDays++;
+      },
+      () => {
+        this.characterService.characterState.status.stamina.value -= 1000;
+        //it still succeeds if you don't have enough stamina - expected to have a good home by now
+        let lowStat = "strength" as AttributeType;
+        let highStat = "strength" as AttributeType;
+        for (const attribute of ["toughness","speed","intelligence","charisma"] as AttributeType[]){
+          if (this.characterService.characterState.attributes[attribute].value < this.characterService.characterState.attributes[lowStat].value) {
+            lowStat = attribute;
+          }
+          if (this.characterService.characterState.attributes[attribute].value > this.characterService.characterState.attributes[highStat].value) {
+            highStat = attribute;
+          }
+        }
+        let statSum = 0;
+        for (const attribute of ["strength","toughness","speed","intelligence","charisma"] as AttributeType[]){
+          if (attribute === lowStat) this.characterService.characterState.increaseAttribute(attribute, 300);
```

Comment:
I don't think you're aware, but this is multiplied by the aptitude multiplier. These numbers are usually 1 or less... @immortalityidle I feel like you could potentially manually enter parts of this PR but as it is there's a lot of needless additions and there's the comment about a lack of testing.